### DISCUSSION TITLE
Dispatch an event on click of a column value

### DISCUSSION THREAD

**Discussion by vickybml** [Power-Components/livewire-powergrid#1235] [BUG]
On click of an cell value, We need to open a modal. We tried by adding a custom column using the below code:
           
            ->addColumn('note', function (ClubAccount $model) {
                $lnote = Note::Where('entity_type', 0)->where('entity_id', $model->id)->orderBy('created_at', 'DESC')->first();
                if (!empty($lnote)) {
                } else {
                    return '<a href="'.'$dispatch(\'showModal\', \'shared.modals.common-dispaly-any-message\', {\'title\':'.$model->id.', \'message\':\'Testing pop up!\'}, \'modal-sm\')'.'">_</a>';
                }
            })
But its not working. Please help.
Thanks.

**Comment by lozynskiadam** [comment]
I am not pretty sure this way but i think your clickable element should refer to js Livewire instance. It should  look something like this:
`<button onclick="Livewire.dispatch('showModal')">...</button>`
